notebooks/en/enterprise_dedicated_endpoints.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>I think it makes more sense to nest this under <code>Creating your first endpoint</code> since they're related topics. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>